stepper.scad

```diff
@@ -194,36 +231,38 @@ function motorLength(model=Nema23, size=NemaMedium) = lookup(size, model);
 
 module motor(model=Nema23, size=NemaMedium, dualAxis=false, pos=[0,0,0], orientation = [0,0,0]) {
 
-  length = lookup(size, model);
+  motorDef = NemaDefinitions[model];
```

Comment:
It looks like this changes the format of the `model` parameter from the motor definition structure to an array index. This isn't acceptable because it'll break many designs floating around the internet that use this module.